inst/private/python_ipc_native.m

```diff
@@ -65,6 +65,8 @@
                     'import sympy as sp'
                     'from sympy import __version__ as spver'
                     'from sympy import *'
+                    'from sympy.sets.fancysets import *'
+                    'from sympy.sets.sets import *'
```

Comment:
maybe do this below the hypergeometric ones and leave a comment like `# for sets`.